### PR TITLE
New eth2 terms for the glossary

### PR DESCRIPTION
**Pull request by ryancreatescopy**
## Description

With the launch of the genesis block🚀, thought we should add some technical glossary terms for the eth2 stuff as they get used a lot!

### CODE REVIEW

**Comment by ryancreatescopy** [comment]
@djrtwo would be great if you could do a quick accuracy check on this for me! Also, if there are any other common terms you think we could add, that'd be awesome.

File: src/content/glossary/index.md [hunk 1]
```diff
@@ -34,10 +34,20 @@ In [Solidity](#solidity), `assert(false)` compiles to `0xfe`, an invalid opcode,
 
 <DocLink to="/developers/docs/security/" title="Security" />
 
+### attestation {#attestation}
+
+A validator vote for a [Beacon Chain](#beacon-chain) or [shard](#shard) [block](#block). Validators must attest to blocks, signalling that they agree with the state proposed by the block.
```
Comment (wackerow) [inline src/content/glossary/index.md:39]: If I am correct, the spelling of "signalling" vs "signaling" is regional. We only have "en" without any differentiation for British vs American, but correct me if I'm wrong, words like "organization" vs "organisation" on the site tend to be the American spelling?
For consistency would suggest changing to `signaling`

File: src/content/glossary/index.md [hunk 2]
```diff
@@ -144,6 +164,12 @@ A short string of data a user produces for a document using a [private key](#pri
 
 A cryptographic algorithm used by Ethereum to ensure that funds can only be spent by their owners.
 
+### epoch {#epoch}
+
+A period of 32 [slots](#slot) (6.4 minutes) in the [Beacon Chain](#beacon-chain)-coordinated system. [Validator](#validator) [committees](#committee) are shuffled every epoch for security reasons. There's an opportunity at each epoch for the chain to be [finalised](#finality).
```
Comment (wackerow) [inline src/content/glossary/index.md:169]: Same as above here with "finalised"... would suggest changing to `finalized`

**Review by wackerow — COMMENTED** [review]
👍

File: src/content/glossary/index.md (hunk 1 above)
Comment (samajammin) [reply src/content/glossary/index.md:39]: That's a good point, there's quite a mix of English (British vs. American) across the site. Probably makes sense to pick one to go with.

Comment (samajammin) [reply src/content/glossary/index.md:39]: Not sure what the best way to decide, e.g. if [Google Trends data](https://trends.google.com/trends/explore?q=organization,organisation) is an accurate signal of understanding/demand:
Image 2020-12-01 at 9 17 24 AM

**Review by samajammin — DISMISSED** [review]
LGTM

**Comment by ryancreatescopy** [comment]
It pained me to do it, but changed to US English. We also go the. thumbs up from Lakshman so let's merge after review.

**Review by samajammin — APPROVED** [review]

**Pull request merged**